main.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'app'
+
+def display_menu
+  puts 'Please choose an option by entering a number: '
+  CHOICES.each { |key, value| puts "#{key} - #{value.to_s.tr('_', ' ')}" }
+  gets.chomp.to_i
+end
```

Comment:
Nice job so far👍🏼, currently I get an `uninitialized constant` error when I try to run your app😉. Please do well to fix it as this makes it impossible for a visitor to make use of your app and can lead to a poor user experience👍🏼. I think this is because the variable `CHOICES` has not been defined😉.

<img width="606" alt="image" src="https://github.com/eamador1/school-library/assets/64643995/0d90401f-d4d4-4a7e-adaa-294f122cfc44">

Reply:
well resolved! ✔️